### ISSUE TITLE
ng build -prod does  copies assets with javascripts missing 

### ISSUE THREAD

**Issue by maclarensg** [angular/angular-cli#1637]
### System
```
 uname -a
Linux gavin-ubuntu 4.4.0-31-generic #50-Ubuntu SMP Wed Jul 13 00:07:12 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

 ~/Development/ngBRM/dist  cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=16.04
DISTRIB_CODENAME=xenial
DISTRIB_DESCRIPTION="Ubuntu 16.04 LTS"
```

### angular-cli version:

```
 ng version    
Could not start watchman; falling back to NodeWatcher for file system events.
Visit http://ember-cli.com/user-guide/#watchman for more info.
angular-cli: 1.0.0-beta.10
node: 5.12.0
os: linux x64
```

### Problem (How to reproduce)
1) Install jquery and materialize-css, ```npm install jquery materialize-css --save```
2) add them to **angular-cli-build.js**
```
var Angular2App = require('angular-cli/lib/broccoli/angular2-app');

module.exports = function(defaults) {
  return new Angular2App(defaults, {
    vendorNpmFiles: [
      'systemjs/dist/system-polyfills.js',
      'systemjs/dist/system.src.js',
      'zone.js/dist/**/*.+(js|js.map)',
      'es6-shim/es6-shim.js',
      'reflect-metadata/**/*.+(ts|js|js.map)',
      'rxjs/**/*.+(js|js.map)',
      '@angular/**/*.+(js|js.map)',
      'materialize-css/dist/**/*',     
      'jquery/dist/jquery.min.js'
    ]
  });
};
```
3) run ```ng build -prod```, all files from materialize  except javascript  are copied, jquery is missing files too.

4) run ```ng build``` has no issues. Applies to ```ng serve``` in production env too.

**Comment by filipesilva** [comment]
Fixed in #1455.

**Comment by alexsorokoletov** [comment]
@filipesilva I've updated the dev dependency and cli installation to beta 10 and still have same problem with `intl-tel-input` package

**Comment by filipesilva** [comment]
@alexsorokoletov it was fixed in #1455 but not yet released. You can use the preview build by following the instructions in https://github.com/angular/angular-cli#webpack-preview-release-update
